padinfo/padinfo.py

```diff
@@ -1512,30 +1512,26 @@ async def jpdgname(self, ctx, *, search_text):
 
     @commands.command(aliases=["jydl", "jpyt"], usage="<dungeon_name> / <monster_name>")
     async def jpyoutube(self, ctx, *, search_text):
-        """Attempt to link to a YouTube search of a leader in a dungeon"""
+        """Link to a YouTube search of a dungeon, with an option to specify leader"""
         dbcog = await self.get_dbcog()
         db: "DBCogDatabase" = dbcog.database.database
-        if '/' in search_text:
-            texts = search_text.split('/')
-        elif ',' in search_text:
-            texts = search_text.split('/')
-        else:
-            texts = search_text.split(maxsplit=1)
-        if len(texts) < 2:
-            return await ctx.send(f"No monster found. Please provide both a dungeon and a monster. "
-                                  f"Perhaps you meant to use `{ctx.prefix}skyo` or `{ctx.prefix}jpdgname`?")
-        dg_text = texts[0]
-        mon_text = texts[1]
-        dg_qs = await QuerySettings.extract_raw(ctx.author, self.bot, dg_text)
+        if '/' not in search_text:
+            dg_text = search_text
+            monster = None
 
-        monster = await dbcog.find_monster(mon_text, ctx.author.id)
-        if monster is None:
-            return await ctx.send(f"No monster found. This command looks for `/` or `,` as a delimiter, "
-                                  f"maybe try again?")
+        elif '/' in search_text:
```

Comment:
oops I *did* have a bug before where `,` wasn't a valid delimiter, but we *should* support splitting by `,` still, it's easier for people on mobile to type